client/dashboard/sites/index.tsx

```diff
@@ -63,10 +64,12 @@ const getFetchSitesOptions = ( view: View, isRestoringAccount: boolean ): FetchS
 };
 
 function getFetchSiteListParams(
-	view: View
-	// isRestoringAccount: boolean TODO: Add site visibility filtering
+	view: View,
+	// isRestoringAccount: boolean TODO: Add site visibility filtering,
+	siteFilters: DashboardFilters = {}
 ): FetchDashboardSiteListParams {
-	const dataviewFieldToSiteProfileField: Record< string, keyof DashboardSiteListSite > = {
+	// The mapping from Dataview fields to Site Profiles fields.
+	const MAPPED_FIELDS: Record< string, keyof DashboardSiteListSite > = {
```

Comment:
Minor: Do these need to be uppercase? Typically, uppercase is reserved for static, compile-time configurations.